leshan-server-demo/webapp/src/components/resources/ResourceControl.vue

```diff
@@ -93,7 +93,7 @@ export default {
       return this.resourcedef.operations.includes("W");
     },
     executable() {
-      return this.resourcedef.operations.includes("E");
+      return this.resourcedef.operations == ("E");
```

Comment:
1. Any reason to use  `(` `)` ?
2. I think `==` should work but generally I prefer to use `===` by default and use `==` only when conversion is needed.  (See : https://stackoverflow.com/a/3586781/5088764)